soroban-env-common/src/env_val.rs

```diff
@@ -222,6 +222,91 @@ impl<E: Env> TryIntoVal<E, u64> for RawVal {
     }
 }
 
+impl<E: Env> IntoVal<E, RawVal> for u128 {
```

Comment:
`TryFromVal` is missing for u128.

Reply:
Yeah I'm .. about to write a PR that deletes TryFromVal from everything (it's not widely used and seems to just clutter up understanding) so I didn't bother adding it here. If you want it for completeness for the time being I'm happy to add it for now, it just didn't seem worthwhile.

Reply:
You won't be able to call using the client in the SDK functions that return a 128 bit type if the TryFromVal is missing. That's something we'd switch to TryIntoVal when removing TryFromVal. It might be worth adding until that happens.

Reply:
Yeah I noticed this because I wrote test contracts in the SDK that use i128 and u128. The i128 test passed, while the u128 one failed to compile because of this.